cloud_experts_tutorials/cloud-experts-aws-secret-manager.adoc

```diff
@@ -281,7 +281,7 @@ spec:
         secretProviderClass: "my-application-aws-secrets"
   containers:
   - name: my-application-deployment
-    image: k8s.gcr.io/e2e-test-images/busybox:1.29
+    image: k8s.gcr.io/e2e-test-images/busybox:1.30
```

Comment:
yes, busybox  image with tag `1.30` can't be available. also for tag `1.29` , maybe we don't need use tag.

Reply:
Per our slack conversation, I've reverted the busybox changes to stay at 1.29. Let me know how it looks now!